pypesto/petab/importer.py

```diff
@@ -78,6 +81,74 @@ def from_yaml(yaml_config: Union[dict, str],
             output_folder=output_folder,
             model_name=model_name)
 
+    def check_gradients(
+        self,
+        *args,
+        rtol: float = 1e-2,
+        atol: float = 1e-3,
+        mode: Literal = None,
+        objective: Objective = None,
+        objAbsoluteTolerance: float = 1e-10,
+        objRelativeTolerance: float = 1e-12,
+        multi_eps=None,
+        **kwargs
+    ):
+        """Check if gradients match finite differences (FDs)
+        Parameters
+        ----------
+        rtol: relative error tolerance
+        atol: absolute error tolerance
+        mode: unction values or residuals
+        objective: objective function
+        objAbsoluteTolerance: absolute tolerance in sensitivity calculation
+        objRelativeTolerance: relative tolerance in sensitivity calculation
+        multi_eps: multiple test step width for FDs
+
+        Returns
+        -------
+        bool
```

Comment:
I'd prefer using return type hints in the function signature instead of specifying it in the docstring.

Reply:
agreed, call it "match" or something